libs/libc/misc/lib_mutex.c

```diff
@@ -195,9 +140,11 @@ bool nxmutex_is_hold(FAR mutex_t *mutex)
  *
  ****************************************************************************/
 
-int nxmutex_get_holder(FAR mutex_t *mutex)
+pid_t nxmutex_get_holder(FAR mutex_t *mutex)
```

Comment:
why not inline all functions in this file?

Reply:
could do.. but if inlined function generates more code than a branch to the function, it increases code size, especially if it is frequently used. I only inlined functions for which I was sure that the code size doesn't increase

Reply:
but some functions is very simple, but not inline.

Reply:
Inlined all the functions which didn't increase image size now.